swyft/inference/posteriors.py

```diff
@@ -47,7 +49,7 @@ def sample(self, N, obs, device=None, n_batch=10_000):
 
             weights[k] = np.exp(val)
 
-        return dict(params=v, weights=weights)
+        return dict(v=v, weights=weights)
 
     def rejection_sample(
```

Comment:
[] This should be moved over to Posteriors rather than living in PosteriorCollection

Reply:
this meaning rejection_sample